src/applications/personalization/profile-2/sass/profile-sidenav.scss

```diff
@@ -0,0 +1,149 @@
+  @include media-maxwidth($medium-screen - 1) {
+    display: none;
+    height: 100%;
+    left: 0;
+    overflow: scroll;
+    padding: 1.6rem;
+    position: absolute;
+    top: 0;
+    visibility: hidden;
+    width: 100%;
+
+    // The overlay that covers the entire page behind the open menu
+    &::before {
+      background: $color-link-default-hover;
+      content: '\00A0';
+      display: block;
+      height: 100%;
+      left: 0;
+      position: fixed;
+      top: 0;
+      width: 100%;
+      z-index: 1;
+    }
+
+    // The actual background behind the side nav menu on mobile
+    > div {
+      -webkit-transition: 100ms transform linear;
+      background: $color-white;
+      border-left: 1px solid $color-gray-light;
+      height: 100%;
+      max-width: 30rem;
+      overflow: auto;
+      padding: units-px(4) units-px(2) units-px(2);
+      position: absolute;
+      right: 0;
+      top: 0;
+      transform: translateX(100%);
+      transition: 100ms transform linear;
+      visibility: visible;
+      width: 80%;
+      z-index: 2;
+    }
+
+    &--opened {
+      display: block;
+      position: fixed;
+      visibility: visible;
+      z-index: $top-layer;
+
+      & > div {
+        transform: translateX(0);
+      }
+    }
+  }
+
+  button.va-sidebarnav-close {
+    position: absolute;
+    right: 2px;
+    top: 2px;
+
+    &:hover,
+    &:focus {
+      background-color: $color-gray-lightest;
+    }
+
+    @include media($medium-screen) {
+      display: none;
+    }
+  }
+
+  h4 {
+    border: 1px solid $color-gray-lighter;
+    border-left: none;
+    border-right: none;
+    color: $color-black;
+    margin-bottom: units-px(2);
+    margin-top: units-px(4);
+    padding: $nav-item-padding;
+  }
+
+  ul {
+    padding: 0;
+  }
+  li {
+    list-style: none;
+    margin-bottom: 0;
+
+    a {
+      border-left: 4px solid transparent;
+      display: inline-block;
+      padding: $nav-item-padding;
+      text-decoration: none;
+      width: 100%;
+
+      &:hover,
+      &:focus {
+        background-color: $color-primary-alt-lightest;
+        border-color: $color-link-default;
+        padding-left: 14px;
+        transition: background-color 0.1s ease-in-out,
+          border-color 0.1s ease-in-out, padding 0.1s ease-in-out;
+      }
+      &.is-active {
+        background-color: $color-gray-lightest;
+        border-left: 4px solid $color-base;
+        color: $color-black;
+        font-weight: bold;
+        padding-left: units-px(1.5);
+      }
+    }
+  }
+}
+
+// This is the trigger-button on mobile
+.va-btn-sidebarnav-trigger {
+  background-color: $color-primary;
+  color: $color-white;
+  margin: units-px(1) 0;
+  width: 100%;
+
+  &:hover {
+    background-color: $color-primary-darker;
+  }
+
+  &:focus,
+  &:hover {
+    svg {
+      fill: $color-white;
+    }
+  }
+
+  &.fixed-trigger {
+    margin: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: $top-layer;
+  }
+}
```

Comment:
This looks a lot like the sidebars that show up at +3 levels deep - https://staging.va.gov/disability/eligibility/hazardous-materials-exposure/agent-orange/related-diseases/.

We have a site-wide SCSS file that imports the style at `src/platform/site-wide/sass/style.scss`. Can we structure the profile's sidebar's HTML to match the site-wide one to mooch off of that?

Reply:
I thought about that but honestly I really hate the idea of hacking the DOM markup artificially to take advantage of existing CSS rules. I'm also a little leery of tightly coupling the look of this new mobile sidenav to the existing sidenav. Although this top level certainly looks like the existing nav's third level, that might change. As it stands it's already different in the fact that in this new sidenav the top-level items are not accordions but are real menu items. But I'll take a look at what would be required to make that happen.

Reply:
I mean I both hate the idea of duplicating CSS that already exists, but I also don't trust CSS to be written in a way that's not brittle so I like the idea of "starting fresh" with a more simplified CSS doc.

Reply:
Looked into this a bit. I don't think that the scss file you pointed to is responsible for the styling of the sidenav at https://staging.va.gov/disability/eligibility/hazardous-materials-exposure/agent-orange/related-diseases/ Just looking at how the "active" menu item is styled with styles targeting `.va-sidebarnav .usa-sidenav-list a.usa-current`. Even if I use the entire `src/platform/site-wide/sass/style.scss` file in my new app, I do not get styles like that.

Reply:
I pulled a lot of my styling out of https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/b0e8508b7257d248663bb4a9a72039762bfb5783/packages/formation/sass/modules/_m-nav-sidebar.scss. But those styles do not appear to get pulled into the main site for some reason.

Reply:
wait. idea...

Reply:
right now I remember. I tried pulling in `@import "~@department-of-veterans-affairs/formation/sass/modules/m-nav-sidebar";` to my app and that blows up due to bad image references that are in that sass module and honestly it was way more trouble than it was worth to try to untangle all of that stuff which is why I just pulled out the bits that actually applied to my sidebar

Reply:
Oh you're right, looks like there's a sidenav SCSS imported in static-pages. That makes sense actually, because sidenavs aren't site-wide....although I wonder what the site-wide one is lol. Anyway this could be useful, or could just be another layer on top of the site-wide one,   `src/applications/static-pages/sass/modules/_m-drupal-sidebarnav.scss`

Reply:
And even if I could get that module to compile in my app, which is possible, it ends up adding so much extra CSS that I do not need. And I need to customize it to make it work anyway. Sorry I think that duplicating some of this code is probably the best of all the bad choices.

Reply:
There's another one actually - `src/platform/site-wide/sass/modules/_m-side-nav.scss`. I think that one plus the Drupal one produces the one I linked to. For some reason our site-wide SCSS imports that instead of the Formation one 🤷